server/planning/commands/populate_planning_types_test.py

```diff
@@ -11,13 +11,12 @@
 import os
 import json
 
-from .populate_planning_types import PopulatePlanningTypesCommand
```

Comment:
You can remove this test file as the command no longer exists, and because tests for AppPopulateCommand are in superdesk-core.

Reply:
Hi @MarkLark86 , I left it here just to test population of `planning_types`, in the core I can't really do it, because this service is a part of planning component.